packages/@okta/vuepress-site/docs/reference/api/apps/index.md

```diff
@@ -6361,3 +6861,214 @@ Property details
 
  * `url` can't have query or fragment parameters.
  * `index` has to be a non-negative number and cannot be duplicated in a set of ACS endpoints configured for an app.
+
+### Provisioning Connection object
+
+Defines the application provisioning connection.
+
+#### Example
+```json
+{
+    "authScheme": "TOKEN",
+    "status": "DISABLED",
+    "_links": {
+        "activate": {
+            "href": "https://${yourOktaDomain}/api/v1/apps/${applicationId}/connections/default/lifecycle/activate",
+            "hints": {
+                "allow": [
+                    "POST"
+                ]
+            }
+        },
+        "self": {
+            "href": "https://${yourOktaDomain}/api/v1/apps/${applicationId}/connections/default",
+            "hints": {
+                "allow": [
+                    "POST",
+                    "GET"
+                ]
+            }
+        }
+    }
+}
+```
+
+#### Provisioning Connection properties
+
+| Property         | Description                                                  | DataType                                                                    | Nullable | Unique | Readonly | Default |
+| ---------------- | ------------------------------------------------------------ | --------------------------------------------------------------------------- | -------- | ------ | -------- | --------- |
+| authScheme              | defines how to  connection is authenticated     | `TOKEN`, `UNKNOWN`                                                           | FALSE    | FALSE  | FALSE    |           |
```

Comment:
Capitals to start the descriptions for all entries in this table

"defines how to connection is authenticated" is missing information - it doesn't make sense

Reply:
> Capitals to start the descriptions for all entries in this table

For the most part I saw properties start with either lowercase or uppercase. Even within sections it can change ex: https://developer.okta.com/docs/reference/api/apps/#list-users-assigned-to-application vs https://developer.okta.com/docs/reference/api/apps/#update-application-credentials-for-assigned-user. I will change to capitals for mine

Reply:
They really all should be changed - that's the standard. I'm not sure this doc has ever been thoroughly reviewed before. Maybe a job for the next app platform intern to fix all these tables?

Reply:
yeah could be a good idea. plus with CIAM efforts it would be good to have consistent API docs